### ISSUE TITLE
libm directory is missing from 0.1.97 crates.io release

### ISSUE THREAD

**Issue by nicholasbishop** [rust-lang/compiler-builtins#543]
When trying to build rust with 0.1.97, I get this error:
```
error: couldn't read /home/nicholasbishop/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.97/src/../libm/src/math/mod.rs: No such file or directory (os error 2)
 --> /home/nicholasbishop/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.97/src/math.rs:3:1
  |
3 | mod libm;
  | ^^^^^^^^^
```

It looks like the release is missing the `libm` directory; I can see it in 0.1.96 but not 0.1.97:
* https://docs.rs/crate/compiler_builtins/0.1.96/source/
* https://docs.rs/crate/compiler_builtins/0.1.97/source/

**Comment by Amanieu** [comment]
Fixed in 0.1.98.

**Comment by nicholasbishop** [comment]
Thanks :)
